.github/workflows/tests.yaml

```diff
@@ -80,15 +80,28 @@ jobs:
         run: |
           sudo apt update
           sudo apt install -y python3-pip python3-setuptools python3-wheel python3-venv libapt-pkg-dev
-      - name: Install external dependencies with homebrew
+      - name: Install skopeo (mac)
```

Comment:
These changes prevent the Jammy builders from getting a Homebrew-installed version of Python 3.12 that has weird behaviour.